AlertaDengue/api/db.py

```diff
@@ -361,7 +361,9 @@ def get_gender_dist(self, db_engine: Engine = DB_ENGINE) -> pd.DataFrame:
 
         with db_engine.connect() as conn:
             result = conn.execute(sql)
-            df_gender_dist = pd.DataFrame(result.fetchall())
+            df_gender_dist = pd.DataFrame(
+                result.fetchall(), columns=result.keys()
+            )
 
             return df_gender_dist.set_index("category", drop=True)
```

Comment:
this line throws an error when the results return no data, added the columns to the dataframe contain the column "category" even if the df is empty